web-services-app/src/main/java/edu/unc/lib/boxc/web/services/rest/modify/ViewSettingController.java

```diff
@@ -0,0 +1,68 @@
+package edu.unc.lib.boxc.web.services.rest.modify;
+
+import edu.unc.lib.boxc.auth.api.Permission;
+import edu.unc.lib.boxc.auth.api.models.AccessGroupSet;
+import edu.unc.lib.boxc.auth.api.services.AccessControlService;
+import edu.unc.lib.boxc.model.api.exceptions.InvalidOperationForObjectType;
+import edu.unc.lib.boxc.model.api.ids.PID;
+import edu.unc.lib.boxc.model.api.objects.RepositoryObjectLoader;
+import edu.unc.lib.boxc.model.api.objects.WorkObject;
+import edu.unc.lib.boxc.model.api.rdf.CdrView;
+import edu.unc.lib.boxc.model.fcrepo.ids.PIDs;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.Resource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static edu.unc.lib.boxc.auth.fcrepo.services.GroupsThreadStore.getAgentPrincipals;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Controller
+public class ViewSettingController {
+    private static final Logger LOG = LoggerFactory.getLogger(ViewSettingController.class);
+    @Autowired
+    private AccessControlService accessControlService;
+    @Autowired
+    private RepositoryObjectLoader repositoryObjectLoader;
+
+    @CrossOrigin
```

Comment:
We shouldn't need cross origin here, it's just going to be the DCR calling this endpoint, whereas with the iiif stuff we would expect it to be called by other websites